code/go/internal/validator/spec.go

```diff
@@ -212,6 +212,7 @@ func (s Spec) rules(pkgType string, rootSpec spectypes.ItemSpec) validationRules
 		{fn: semantic.ValidateKibanaFilterPresent, since: semver.MustParse("3.0.0")},
 		{fn: semantic.ValidateKibanaNoLegacyVisualizations, types: []string{"integration"}, since: semver.MustParse("3.0.0")},
 		{fn: semantic.ValidateDimensionsPresent, types: []string{"integration"}, since: semver.MustParse("3.0.1")},
+		{fn: semantic.ValidateCapabilitiesRequired, since: semver.MustParse("2.10.0")}, // capabilities definition was added in spec version 2.10.0
```

Comment:
I set that version to ensure that the expected capabilities are added into the manifest since the first version were those fields were introduced into the manifest (2.10.0).

We could also start testing this validation from 3.0.0. That spec was the minimum one set for Serverless projects. For instance, for observability: 
https://github.com/elastic/kibana/blob/68b3267ca2f76d8de4fedef622608d8685e7eceb/config/serverless.security.yml#L78

The current packages defining `security-rule` assets are:
- `security_detection_engine`: defines spec version 3.0.0
- `tychon`: defines spec version 3.2.2

Currently, there are no other packages failing in integrations because of that.
Do you think it would be better to set 3.0.0 to match with versioning used in Serverless rather than 2.10.0?